Ascend.java

```diff
@@ -1,7 +1,28 @@
 // Generates three integer random numbers in a given range,
 // and prints them in increasing order.
+
+import java.util.Random;

```

Comment:
This is a violation of the HW Guidelines.
you may only use material taught in lectures & recitations. 
Next time, it will cause deduction of points.